packages/app-store/utils.ts

```diff
@@ -104,6 +104,16 @@ function getApps(credentials: CredentialDataWithTeamName[], filterOnCredentials?
   return apps;
 }
 
+export function getAppsBySlug(
```

Comment:
If we want to get a specific app instead of all, can be useful for v2 api.

Reply:
Nit: getAppBySlug since we're returning a singular App instead of multiple